### PR TITLE
[IMP] Update Travis to add Transifex conf

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Comment by ghost** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,28 @@
...
+  - VERSION="7.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - TRANSIFEX_USER='transbot@odoo-community.org'
+  - secure: DncQWtzMNwjo/k6Kmotl5IXI1l7sj3AcFTPSjT8i5D/2C42Y2bYTahdXcabDIXhBeUPrMxiQFc/A4cxoDPyn8u2YeLpuX4NNiUtTOIuI0GU3zanna7AUO31JGwMle4Rzaepa3lHxFaQzqs9+auawlt1s9qmwmRsfYpjyclf0ydw=
+
+  matrix:
+  - LINT_CHECK="1"
+  - TRANSIFEX="1"
   - VERSION="7.0" ODOO_REPO="odoo/odoo" EXCLUDE="framework_agreement,purchase_landed_costs"
   - VERSION="7.0" ODOO_REPO="OCA/OCB" EXCLUDE="framework_agreement,purchase_landed_costs"
   - VERSION="7.0" ODOO_REPO="odoo/odoo" INCLUDE="framework_agreement"
```
Comment (Ehtaga) [inline .travis.yml]: Didn't follow all the debate about travis variables, but aren't you supposed to add TESTS="1" on your tests? Such as in https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml

And VERSION is unnecessary as you defined it in `global`

**Comment by pedrobaeza** [comment]
@Ethaga is right. I didn't notice due to I see only diff. Thanks for noticing.

**Comment by Ehtaga** [comment]
:+1:

**Comment by max3903** [comment]
purchase_landed_costs fails as before.

@pedrobaeza can you merge ?

**Comment by pedrobaeza** [comment]
Merging

**Pull request merged**